tokenizer/schemas.py

```diff
@@ -22,6 +22,12 @@ class TokenizeTeiRequestSchema(Schema):
         required=True,
         description=gettext("Language code of text to be tokenized.")
     )
+    direction = fields.Str(
+        required=False,
+        description=gettext("Text direction."),
+        missing = "ltr",
+        validate = validate.OneOf(["ltr","rtl"])
+    )
```

Comment:
it's not really used at the moment, but it's in the api in case we need it in the future.